api.ts

```diff
@@ -498,7 +500,22 @@ export function makeParameter<T extends ParameterType, O extends ParameterOption
     autocomplete = wrapMetadataFunction(autocompleteDefOrItems);
   }
 
-  return Object.freeze({...rest, autocomplete, type: actualType}) as ParamDefFromOptionsUnion<T, O>;
+  let crawlStrategy: CrawlStrategy | undefined;
+  if (crawlStrategyDef) {
+    if (crawlStrategyDef.parentTable) {
+      const {tableName, propertyKey} = crawlStrategyDef.parentTable;
+      crawlStrategy = {
+        parentTable: {
+          tableName,
+          propertyKey: normalizeSchemaKey(propertyKey),
+        },
+      };
+    } else {
+      crawlStrategy = crawlStrategyDef;
```

Comment:
Curious if this `else` will ever be hit? when would `crawlStrategyDef` be defined but not `parentTable`?

Reply:
Not in this PR, but my plan is to add date range crawling, etc into `crawlStrategy` (that's why I defined it at the param level)